spyder/utils/palette.py

```diff
@@ -78,6 +78,12 @@ class SpyderPaletteDark:
     SPYDER_LOGO_WEB = Logos.B40
     SPYDER_LOGO_SNAKE = Logos.B50
 
+<<<<<<< HEAD
+    # Colors for disabled state toolbar buttons
+    COLOR_DISABLED = ICON_1, Opacity.OPACITY_DISABLED
+
+=======
+>>>>>>> parent of 1373e8594... Add constant and argument for disabled toolbar buttons
```

Comment:
Hey @isabela-pf, you left this unnecessary code after you reverted your changes. Please remove it and after that merge your branch with `5.x` to make our tests pass.

Thanks!